tests/Liquid/StandardFiltersTest.php

```diff
@@ -411,6 +411,8 @@ public function testSlice()
 			}
 			$this->assertEquals($item[1], $actual);
 		}
+
+		$this->assertEquals('Владимир', StandardFilters::slice('Владимир', 0, 15));
```

Comment:
Will that work?
```suggestion
		$this->assertEquals('Владимир', StandardFilters::slice('Владимир Владимирович', 0, 15));
```

Reply:
No, on old version test failed:
```
Failed asserting that two strings are equal.
Expected :'Владимир'
Actual   :'Владими�'
```
Because `substr` split string per byte, and Russian characters take 2 bytes.

Reply:
@sanmai @PNixx the test is not ideal and not easy to understand.
The test will be fail if slice dont use multi-byte safe substr. But nobody know this. The string has a length of 8 chars why cut a length of 15? 

I think this modification everyone should understand and it also fails in the older version:
```suggestion
		$this->assertEquals('Владимир', StandardFilters::slice('Владимир Владимирович', 0, 8));
```